client/src/commonMain/kotlin/com/algolia/search/configuration/Region.kt

```diff
@@ -22,9 +22,23 @@ public sealed class Region {
         override fun toString(): String = raw
     }
 
+    /**
+     * Available regions, used in [ConfigurationPersonalization].
+     */
+    public sealed class Personalization(override val raw: String) : Raw<String> {
+
+        public object EU : Personalization(KeyEU)
+        public object US : Personalization(KeyUS)
+        public class Other(override val raw: String) : Personalization(raw)
+
+        override fun toString(): String = raw
+    }
+
     /**
      * Available regions, used in [ConfigurationRecommendation].
      */
+    @Deprecated("use Recommendation instead", replaceWith = ReplaceWith("Personalization"))
```

Comment:
Is it possible to replace this class with a typealias as you did with `ClientRecommendation` and `ConfigurationRecommendation`?

Reply:
Unfortunately it is not possible to introduce nested `typealias`, only top level declarations are supported ATM :/